src/generator/AutoRest.CSharp/CodeNamerCs.cs

```diff
@@ -80,6 +80,10 @@ public override string EscapeDefaultValue(string defaultValue, IModelType type)
                 {
                     return "new " + type.Name + "()";
                 }
+                if (type is EnumType && (type as EnumType).ModelAsString)
```

Comment:
Did you do a test run with the compare script? I'm nervous about what happens on all the generators...